core/src/test/scala/precog/RunnerSpec.scala

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package precog
+
+import cats.effect.IO
+import sbt.file
+
+import scala.collection.immutable.Seq
+import scala.sys.process.ProcessLogger
+
+class RunnerSpec extends org.specs2.mutable.Specification {
+  val log = sbt.util.LogExchange.logger("test")
+
+  "Fluent builders" should {
+    "be equivalent to case class build" in {
+      val runner = Runner[IO](log)
+      runner === Runner[IO](log, false, None, Map.empty, Seq.empty)
+      runner.stderrToStdout === Runner[IO](log, merge = true)
+      runner.cd(file("./target")) === Runner[IO](log, workingDir = Some(file("./target")))
+      runner.withEnv("X" -> "Y", "W" -> "Z") === Runner[IO](log, env = Map("X" -> "Y", "W" -> "Z"))
+      runner.hide("password") === Runner[IO](log, hide = Seq("password"))
```

Comment:
We strongly prefer a `mustEqual` style due to the fact that `===` is a Cats equality operator.